jbpm-task-assigning/jbpm-task-assigning-core/src/test/java/org/jbpm/task/assigning/model/solver/TaskHelperTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.task.assigning.model.solver;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+import org.jbpm.task.assigning.model.Group;
+import org.jbpm.task.assigning.model.OrganizationalEntity;
+import org.jbpm.task.assigning.model.Task;
+import org.jbpm.task.assigning.model.User;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.jbpm.task.assigning.model.solver.TaskHelper.isPotentialOwner;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class TaskHelperTest {
+
+    private static final int SIZE = 10;
+    private static final Random RANDOM = new Random();
+
+    private List<User> availableUsers;
+    private List<Group> availableGroups;
+    private Task task;
+
+    @Before
+    public void setUp() {
```

Comment:
After reading through this test case I don't think a list of 10 user and 10 groups is needed - neither is using the `Random` to pick one of them.
Having just 2 users and 2 groups would be sufficient - without keeping them in collections and picking them randomly.
Please see first my comment about eventually getting rid of the `TaskHelper` class.

Reply:
was changed to 2